attestation-container/go.mod

```diff
@@ -0,0 +1,16 @@
+module microsoft/attestation-container
+
+go 1.19
+
+require (
+	golang.org/x/sys v0.3.0
```

Comment:
Will this be auto-upgraded by dependendabot? If not, is there a way to make those ">="?

Reply:
It supports golang.
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates

I'll add `gomod` to .github/dependabot.yml

Reply:
Added [663b656](https://github.com/microsoft/CCF/pull/4694/commits/663b65664a0e76fd34ccf7f96263606f8db36152)